sktime/classification/tests/test_all_classifiers.py

```diff
@@ -86,13 +86,20 @@ def test_classifier_output(self, estimator_instance, scenario):
 
         # check predict
         assert isinstance(y_pred, np.ndarray)
-        assert y_pred.shape == (X_new_instances,)
+        assert y_pred.shape == (X_new_instances,) or y_pred.shape == (
+            X_new_instances,
+            y_pred.ndim,
```

Comment:
this does not look right, `ndim` is 1 or 2 always.

Reply:
That has to be `y_train.shape[1]` iff `y_train.ndim > 1`. I hope I got it.

Reply:
This change is yet to be done.